clusters/app.ci/supplemental-ci-images/azure/base-images/ci-base.yaml

```diff
@@ -42,16 +41,15 @@ items:
             lsof \
             make \
             net-tools \
-            PyYAML \
```

Comment:
This seems to only be used by the arcconformance jobs which are disabled about a year.